lib/web_ui/test/canvaskit/h5vcc_test.dart

```diff
@@ -53,7 +53,7 @@ void testMain() {
     });
 
     test('Surface acquireFrame uses getH5vccSkSurface', () {
-      final Surface surface = SurfaceFactory.instance.getSurface();
+      final Surface surface = SurfaceFactory.instance.getSurface()!;
```

Comment:
The fact that we never had to do this once in the embedded_views.dart, blows my mind :P

Reply:
It looks like `getSurface` is only used in tests now. I should be able to get rid of it, but I need to move the `printWarning` to `embedded_views.dart`